.travis.yml

```diff
@@ -21,7 +21,7 @@ os:
   - linux
 
 go:
-  - "1.12"
+  - "1.15"
```

Comment:
Most of dev env is still in 1.12.
Any particular reason to move to 15?

Reply:
Hi @sunilgovind  According to https://endoflife.date/go, v1.12 was released 1.5 years ago and it is marked as EOL.